src/controllers/favorites.controllers.js

```diff
@@ -29,8 +29,20 @@ export const findAllFavorites = async () => {
   }
 };
 export const deleteFavorite = async (data) => {
```

Comment:
Why not delete a favourite by its `id`? `id` should be available on the front-end and a typical delete request would be sent to `api/favorite/:favoriteId` and the `id` can be obtained from a URI. No request body is needed and if you want make sure a proper favourite is deleted, you need to identify only a user and the user can be identified once the authorisation is in place and this action can only be done by logged in users so a user identification is guaranteed and even without this identification it is unlikely that a random favourite will get deleted instead if someone intended malicious activity because the id of a favourite is `UUID` which is not easy to find by chance.